.github/workflows/Release-helm-dnsmasqaks.yml

```diff
@@ -21,12 +21,15 @@ jobs:
         git config user.name "$GITHUB_ACTOR"
         git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
-    - name: Get the version
-      id: vars
-      run: echo ::set-output name=tag::$(echo main-${GITHUB_SHA})
-      
-    - name: Echo Docker images tag
-      run: echo ${{steps.vars.outputs.tag}}
+    - name: Setup Helm
+      uses: azure/setup-helm@v3.5 
+      with:
+        version: v3.9.2
+    
+    - name: Add Helm dependencies
```

Comment:
nit: Add Helm dependencies -> Add Helm chart repositories?

Reply:
Done in all current chart workflows.